modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorInputs.es.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function ContributorInputs({contributors}) {
+	return contributors.map(
+		(criteria, i) => {
+			return (<React.Fragment key={i}>
```

Comment:
`<React.Fragment` should be on the next line. Especially so that it lines up with the closing tag.

```jsx
return (
     <React.Fragment>
          //...
     </React.Fragment>
)
```